### PR TITLE
feat: add cron job to restart the service every few days

### PR DESCRIPTION
**Pull request by akhileshthite**
https://github.com/hyphacoop/distributed-press-organizing/issues/95
### Added cron job to restart api.distributed.press service every 3 days

```sh
sudo ansible-playbook distributed_press.yml -i local_deploy.yml --tags cron
```
Screenshot 2023-10-10 at 11 39 03 PM

```sh
sudo crontab -l
```
Screenshot 2023-10-10 at 11 39 40 PM

### CODE REVIEW

**Comment by akhileshthite** [comment]
> [hyphacoop/distributed-press-organizing#95](https://github.com/hyphacoop/distributed-press-organizing/issues/95)
> 
> ```
> sudo ansible-playbook distributed_press.yml -i local_deploy.yml --tags cron
> ```
> 
> Screenshot 2023-10-10 at 11 39 03 PM
> 
> ```
> sudo crontab -l
> ```
> 
> Screenshot 2023-10-10 at 11 39 40 PM

@RangerMauve 
I don't have a linux machine to test the `systemctl` command.
However, I adjusted the cron timing every minute for quicker testing to ensure the `systemctl` command works as expected.

Can you test this?
```
- name: "Test cron job to restart api.distributed.press service every minute"
  cron:
    name: "Restart api.distributed.press service"
    minute: "*"
    job: "/bin/systemctl restart {{ distributed_press_service_name }}"
  become: yes
  tags:
    - cron
```

```
sudo ansible-playbook distributed_press.yml -i local_deploy.yml --tags cron
```

File: ansible/roles/distributed_press/templates/distributed_press_restart_cron.j2 [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Restart the api.distributed.press service every few days
+* * */3 * * /bin/systemctl restart {{distributed_press_service_name}}
```
Comment (RangerMauve) [inline ansible/roles/distributed_press/templates/distributed_press_restart_cron.j2:2]: Is this file still needed?

**Review by RangerMauve — CHANGES_REQUESTED** [review]
Tested and it works. Mind checking if the template is still needed and removing it if not?

File: ansible/roles/distributed_press/templates/distributed_press_restart_cron.j2 (hunk 1 above)
Comment (akhileshthite) [reply ansible/roles/distributed_press/templates/distributed_press_restart_cron.j2]: No, removed.

**Review by RangerMauve — APPROVED** [review]

**Pull request merged**